pom.xml

```diff
@@ -189,9 +191,52 @@
         <artifactId>maven-release-plugin</artifactId>
         <version>2.5.3</version>
       </plugin>
+    
+    <plugin>
+  <groupId>org.eclipse.jetty</groupId>
+  <artifactId>jetty-maven-plugin</artifactId>
+  <version>9.2.22.v20170606</version>
+  <configuration>
+          <scanIntervalSeconds>0</scanIntervalSeconds>
+          <stopPort>11079</stopPort>
+          <stopKey>STOP</stopKey>
+          <httpConnector>
+            <port>${playbackServerPort}</port>
+          </httpConnector>
+       
+        </configuration>
+        
+</plugin>
     </plugins>
```

Comment:
The formatting of this xml seems to be off can you reformat to line up the begin and end tags?